curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-set-class.english.md

```diff
@@ -6,19 +6,26 @@ challengeType: 1
 
 ## Description
 <section id='description'>
-In the next few exercises we are going to create a function to emulate a data structure called a "Set". A Set is like an array, but it cannot contain duplicate values. The typical use for a Set is to simply check for the presence of an item. This can be implemented with an object, for instance:
+In this exercise we are going to create a class Set to emulate a data structure called a "Set". A Set is like an array, but it cannot contain duplicate values. The typical use for a Set is to simply check for the presence of an item. This can be implemented with an object, for instance:
 <blockquote>var set = new Object();<br>set.foo = true;<br>// See if foo exists in our set:<br>console.log(set.foo) // true</blockquote>
-In the next few exercises, we will build a full featured Set from scratch.
-For this exercise, create a function that will add a value to our set collection as long as the value does not already exist in the set. For example:
-<blockquote>this.add = function(element) {<br>  //some code to add value to the set<br>}</blockquote>
-The function should return <code>true</code> if the value is successfully added and <code>false</code> otherwise.
+
+First, we will create a method that will add a value to our set collection as long as the value does not already exist in the set.
+
+Then we will create a delete/remove method for our set that will accept a value, and if this value exists inside the set collection, this method will remove it and return true, whereas it will return false if the value does not exist inside the set collection.
+
+And finally, we will create a size method for our Set class. This method will return the number of elements inside the set collection.
 </section>
 
 ## Instructions
 <section id='instructions'>
+Create a new <code>add</code> method for class Set that should add a value to the set and return <code>true</code> if the value was successfully added to the set and <code>false</code> otherwise.
```

Comment:
I also tried to simplify these instructions a bit more by removing unnecessary words:

```
Create an <code>add</code> method that adds a unique value to the set collection and returns <code>true</code> if the value was successfully added and <code>false</code> otherwise.

Create a <code>remove</code> method that accepts a value and checks if it exists in the set. If it does, then remove that value from the set collection and return <code>true</code>. Otherwise, return <code>false</code>.

Create a <code>size</code> method that returns the size of the set collection.
```

Reply:
Thanks again! I've made the suggested changes but I'm a little bit confused about the last sentence (made it **bold**) in <code>remove</code> method, 
> Create a <code>remove</code> method that accepts a value and checks if it exists in the set. **If it does, then remove that value from the set collection and return <code>true</code>. Otherwise, return <code>false</code>.**

I think it should be like this because we are talking about the <code>remove</code> method but I'm not sure which one will be better for the reader to understand. Can you please take a look at this and suggest changes that will make it a bit easier to understand? Thank you! - 
```
If it does, then this method should remove it from the set collection, and return <code>true</code>. Otherwise, it should return <code>false</code>.
```

Reply:
@abhishekakade, thank you for making all those changes! And you're right, what I originally wrote for the remove method could be confusing. The way you have it now is perfectly fine, so let's leave it the way it is.

Reply:
Yes, thank you so much for all the suggestions! I'll update the PR.